vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -106,6 +112,10 @@ company and technology should be excluded from the analysis. At this point, it
 is strictly recommended to set this to TRUE (default) because there is not yet a
 dedicated mechanism in place that handles companies which are phasing out
 certain technologies within the time frame of the production forecast.
+* `return_results` Logical parameter that indicates if the outputs of the
+analysis should be directly written to the indicated output path or simply
+returned as objects within the running session. The default is FALSE, which
```

Comment:
- list of results.
- I was wondering if to add a use example here, but then again who wants to use this option should be able to understand it without help.

Reply:
done (and no)